src/Main.c

```diff
@@ -76,12 +79,16 @@ httpse_getopt_long(int argc, char **argv, HttpseRulesetOptions *roptions)
 
 		/* libcurl options */
 		{"capath",          required_argument, NULL, 260},
-		{"compressed",       no_argument,      NULL, 261},
+		{"compressed",      no_argument,       NULL, 261},
 		{"connect-timeout", required_argument, NULL, 262},
 		{"max-time",        required_argument, NULL, 'm'},
 		{"tr-encoding",     no_argument,       NULL, 263},
 		{"useragent",       required_argument, NULL, 'A'},
 		{"verbose",         no_argument,       NULL, 'v'},
+		{"tlsv1",           no_argument,       NULL, '1'},
+		{"tlsv1.0",         no_argument,       NULL, '1'},
+		{"tlsv1.1",         no_argument,       NULL, 266},
+		{"tlsv1.2",         no_argument,       NULL, 267},
```

Comment:
Would you be interested to add TLSv1.3 support as well? https://curl.haxx.se/libcurl/c/CURLOPT_SSLVERSION.html

Reply:
Setting minimal version to `CURL_SSLVERSION_TLSv1_3` causes a segmentation fault for some reason.